cn/HowToContributeSCRF_CN

```diff
@@ -0,0 +1,66 @@
+# 怎麼對SCRF做出貢獻
+
+## 什麼是SCRF？
+
+[智能合約研究論壇](https://www.smartcontractresearch.org/)（SCRF）是來自世界各地的學者、研究人員和業界領袖聚集在一起討論研究，徵求peer review，並尋找可以合作的新專案的地方。
+
+區塊鏈研究需要全球的合作、平衡的激勵機制、新的研究、以及編輯的獨立性。
+我們相信該論壇將為學術界、工業界和新興的區塊鏈技術帶來不少幫助，將公正的區塊鏈研究集中在一個單一的、文明的、可信賴的空間，可以加速該領域現有的工作。SCRF能夠激發非常有利的產學合作。
+
+## 我們的研究人員
+我們的核心研究團隊，_Research Fellows_，已經承諾每周至少工作10小時，並參加每週的定期會議。一般來說，研究員們各自代表一個較大的群體，如大學區塊鏈的社團、公司或大學，且通常擁有博士學位或正在攻讀博士學位。他們已經成功地為SCRF發表了許多研究摘要。
+
+較喜歡偶爾工作，且較少參與會議的研究人員則被稱為 _Research Associates_。作為Research Associate，除了在我們的聊天室和論壇中保持一定的存在感，以及偶爾為SCRF提供評論或摘要外，不需要任何條件。完成若干研究總結的Research Associate可能會被邀請成為Research Fellow。
+
+為了使SCRF成為網上最強的區塊鏈研究論壇，我們當然必須對您的時間和專業知識給予獎金。**為SCRF做的所有工作都是有報酬的。**
+
+## 我們正在尋找的人才
+為了彌合工業界和學術界、提出問題者與解決問題者之間的鴻溝，我們需要尋找有能力總結現有的區塊鏈研究的人員。
+
+研究團隊是全論壇的基礎，他們的任務是確保有源源不絕的嶄新研究，能夠提供給同行同業的研究員。他們通過從各種區塊鏈相關領域中挑選出最好的論文，並對其進行總結，使之更容易被理解。
+
+SCRF被設計來揭露最前瞻的研究，並以相較於正式研究發表文件更簡潔的格式呈現出來。我們的目標是統整各種的研究，將各種概念和專有名詞標準化，最終提供給區塊鏈社群一個有吸引力的客觀對談空間。
+
+各種區塊鏈研究機構在出版刊物上需要協助。我們瞭解到，對於鑽研這個領域的研究人員和博士後研究員來說，文獻回顧可能需要相當多的時間和精力。 SCRF將會是一個使學術研究"盈利化"的場域，並將研究內容轉換成摘要格式，與區塊鏈社群共享。
+
+* [目前需要的專業](https://www.smartcontractresearch.org/tag/summary)
+除了研究人員，我們還在尋找版主、編輯、專案經理、社交媒體專家等角色。
+
+## 報酬
+在SCRF，從撰寫研究摘要到審稿、平面設計，再到各種行政職務，有各式各樣的工作要做。我們支援有彈性的付款時程，酬金、補助金或其他安排。
+
+當我們的研究人員更深入地參與專案，或承擔更大的工作量時，我們有更長期的關係可供選擇（包括受薪職位）。
+
+_我們將用加密貨幣支付您的薪水。對於較長的承諾，也可以做出其他安排。_
+
+如果您因為您的大學或資助機構而不能接受直接付款，我們可以討論替代方案。這可能包括：
```

Comment:
This character seems strange as well `：`. Is it also a convention to use that instead of `:` for Chinese translations?